accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/Utils.java

```diff
@@ -234,4 +311,14 @@ private static Field toField(AccioMDL accioMDL, String modelName, Column column,
                 .type(column.getType())
                 .build();
     }
+
+    public static Query createDateSpineQuery(DateSpine dateSpine)
+    {
+        String sql = format("SELECT * FROM UNNEST(GENERATE_TIMESTAMP_ARRAY(TIMESTAMP '%s', TIMESTAMP '%s', %s)) t(metric_time)", dateSpine.getStart(), dateSpine.getEnd(), dateSpine.getUnit().getIntervalExpression());
```

Comment:
Because it's not a common function in SQL, I just chose `GENERATE_TIMESTAMP_ARRAY` to be the name of this feature. For `DUCKDB` dialect, we will format it to the SQL String `GENERATE_SERIES()`. For `Trino`, we may need to format this to other function.